bridge/setu/broadcaster/broadcaster.go

```diff
@@ -40,7 +40,8 @@ type TxBroadcaster struct {
 // NewTxBroadcaster creates new broadcaster
 func NewTxBroadcaster(cliCtx client.Context, cdc codec.Marshaler, flagSet *pflag.FlagSet) *TxBroadcaster {
 	// current address
-	address := hmCommonTypes.BytesToHeimdallAddress(helper.GetAddress())
+	addr := helper.GetAddress()
```

Comment:
I didn't understand this change. The previous line should do the work, isn't it?